lib/module.php

```diff
@@ -191,10 +191,19 @@ trait Hm_Handler_Validate {
      * @return bool
      */
     public function validate_method($session, $request) {
```

Comment:
@Baraka24, the logic looks good, but you could simplify it by providing a clear return after method validation to reduce redundancy. Can you try again?

Reply:
Done Thanks @kambereBr.